qml/pages/MainPage.qml

```diff
@@ -145,6 +152,10 @@ Page {
             PlugHerDe.updatedet(index, trackid, showdet);
             historyModel.set(index, {"typec": "#0091cd"});
         }
+		else if (historyModel.get(index).type == "LAPOSTE") {
```

Comment:
Could you fix the indentation here?

Reply:
Done

> 
> 
> Thanks! I tested this, but I'm not able to get any data from the API. I tried calls for numbers like this (taken from the La Post Twitter mentions, it's often a good source of tracking codes for testing :) ):
> 
>     * 8R42646922018
> 
>     * RH664518443pt
> 
>     * 1L02490640971
> 
>     * LZ474788705CN
> 
>     * 6A20096862948
> 
> 
> Could you test if these work for you to make sure my configuration is correct? I also tried making the call with just cURL, but that didn't work either.

Hi, they do work (supposing you used a production auth key, which I didn't push in the header file), though I found they only supported fr_FR lang (I was getting errors in the emulator because locale was set to english). It's corrected too.
I can send you the auth key by email if you haven't generated one.